test/jasmine/bundle_tests/choropleth_test.js

```diff
@@ -5,24 +5,34 @@ var PlotlyChoropleth = require('@lib/choropleth');
 
 var createGraphDiv = require('../assets/create_graph_div');
 var destroyGraphDiv = require('../assets/destroy_graph_div');
+var failTest = require('../assets/fail_test');
+
+var LONG_TIMEOUT_INTERVAL = 5 * jasmine.DEFAULT_TIMEOUT_INTERVAL;
 
 
 describe('Bundle with choropleth', function() {
     'use strict';
 
     Plotly.register(PlotlyChoropleth);
 
+    var gd;
+
     var mock = require('@mocks/geo_multiple-usa-choropleths.json');
 
-    beforeEach(function(done) {
-        Plotly.plot(createGraphDiv(), mock.data, mock.layout).then(done);
+    beforeEach(function() {
+        gd = createGraphDiv();
     });
 
     afterEach(destroyGraphDiv);
 
-    it('should graph choropleth traces', function() {
-        var nodes = d3.selectAll('g.trace.choropleth');
+    it('should graph choropleth traces', function(done) {
+        Plotly.plot(gd, mock.data, mock.layout)
+        .then(function() {
+            var nodes = d3.selectAll('g.trace.choropleth');
 
-        expect(nodes.size()).toEqual(4);
-    });
+            expect(nodes.size()).toEqual(4);
+        })
+        .catch(failTest)
+        .then(done);
+    }, LONG_TIMEOUT_INTERVAL);
```

Comment:
We've seen this one fail a lot - and this morning seems like circle is super stressed or something, every intermittent test failure we've ever had seems to be biting us today. Choropleth needs to make a request for map data - it's local but still perhaps slow? Just give it some more time before we fail.

Reply:
> it's local but still perhaps slow?

Actually, no. Geo subplots in jasmine tests get their map data from https://cdn.plot.ly/world_110m.json and friends.

You brought this up previously (see https://github.com/plotly/plotly.js/issues/163) but I never got around to fixing that.

----

:ok_hand: for bumping the timeout interval.